### ISSUE TITLE
add analytics code

### ISSUE THREAD

**Issue by ashwini0529** [GDGVIT/Devfest-2.0#15]
```<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-108094768-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-108094768-1');
</script>
```

**Comment by saifat29** [comment]
@ashwini0529 I want to do it.

**Comment by alexfuser** [comment]
Hello @ashwini0529 

I can take it? I'd like to help.

**Comment by technophilic** [comment]
fixed in #16
